html/browsers/browsing-the-web/scroll-to-fragid/scroll-frag-non-utf8-encoded-document.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<title>Fragment Navigation: fragment id should not be found in non UTF8 document</title>
+<meta name=timeout content=long>
+<meta http-equiv="Content-Type" content="text/html; charset=gbk"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<div></div>
+<div id="&#x586f" style="position:absolute; top:100px;"></div>
+<div style="height:200em;"></div>
+<script>
+setup({ single_test: true });
+assert_equals(location.hash, "", "Page must be loaded with no hash");
```

Comment:
Maybe also assert that document.characterSet is GBK?

Reply:
Done.